CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Features
 
+- `[@jest/cli, jest-config, @jest/environment, jest-environment-node, jest-environment-jsdom, @jest/reporters, jest-runtime, @jest/types]` Add `getSeed()` to the jest object available at runtime ([#13400](https://github.com/facebook/jest/pull/13400/))
```

Comment:
Should split up in multiple entries. One for the new config, and one for the `jest` object API